jetson/percep/meson.build

```diff
@@ -7,6 +7,7 @@ all_deps = [opencv, lcm]
 
 with_zed = get_option('with_zed')
 obs_detection = get_option('obs_detection')
+vm = get_option('vm')
```

Comment:
NIT: What does vm stand for? I assume it means virtual machine and would suggest that you are in a virtual machine but that is unclear.

Reply:
You use the build flag when building from a virtual machine, since in order to get the code working in virtualbox I had to downgrade our VTK version from 8.2 to 6.3. We'll migrate over to 6.3, but I didn't want to screw over everyone with 8.2 installed so I added the build flag

Reply:
Where is this documented?

Reply:
Sorry. I have to add it to the README